app/screens/draft_scheduled_post_options/send_draft.test.tsx

```diff
@@ -113,4 +145,85 @@ describe('Send Draft', () => {
         const {getByText} = wrapper;
         expect(getByText('Send')).toBeTruthy();
     });
+
+    test('should handle deleteScheduledPost failure and call handleUpdateScheduledPostErrorCode', async () => {
+        let capturedHandler: Function;
+        const mockError = new Error('Failed to delete scheduled post');
+        jest.mocked(deleteScheduledPost).mockResolvedValue({error: mockError});
+        jest.mocked(sendMessageWithAlert).mockImplementation((params) => {
+            capturedHandler = params.sendMessageHandler;
+            return Promise.resolve();
+        });
```

Comment:
0/5 using the capture handler is one option, but I feel it is more elegant / intention revealing to access the calls of the mock.

Reply:
Done, thanks on this BTW 😄